roles/kubernetes/preinstall/defaults/main.yml

```diff
@@ -34,7 +34,7 @@ resolveconf_cloud_init_conf: /etc/resolveconf_cloud_init.conf
 # All inventory hostnames will be written into each /etc/hosts file.
 populate_inventory_to_hosts_file: true
 
-sysctl_file_path: "/etc/sysctl.d/99-sysctl.conf"
+sysctl_file_path: "{{ sysctl_file_path }}"
```

Comment:
Here also needs to be `sysctl_file_path: /etc/sysctl.d/99-sysctl.conf` because of the same name.

Reply:
Done.